resource/constants.rs

```diff
@@ -108,8 +106,17 @@ pub mod diagnostics {
 
     pub const UNKNOWN_STR: &'static str = "Unknown";
 
-    pub const REPORTING_URI: &str = "https://diagnostics.typedb.com/";
+    pub const SERVICE_REPORTING_URI: &str = "https://diagnostics.typedb.com/";
+    pub const POSTHOG_BATCH_REPORTING_URI: &str = "https://us.i.posthog.com/batch/";
+    // The key is write-only and safe to expose
+    pub const POSTHOG_API_KEY: &str = "phc_pYoyROZCtNDL8obeJfLZ8cP0UKzIAxmd0JcQQ03i07T";
```

Comment:
Alex said that it's a common approach to just use these public keys for pushes, and PostHog also mentions it on their settings page.